### DISCUSSION TITLE
Pairing handling

### DISCUSSION THREAD

**Discussion by mpiffari** [Reedyuk/blue-falcon#128] [Q&A]
Hi,
does this library handle pairing key exchange with BLE device for encrypted communication?

**Comment by Reedyuk** [comment]
Does the native sdks for iOS and Android do this out of the box?
If yes, then yes probably but I have not tested it

**Reply by mpiffari** [reply]
As far as I know, Android can handle natively the paring/bonding procedure.
Instead iOS doesn't offer any system API to use for key pair exchange process.

**Reply by Reedyuk** [reply]
is there any additional changes needed for the android side? obviously ios wouldnt work

**Comment by Reedyuk** [comment]
Bonding is now supported for Android. As for iOS - bonding is handled automatically.
